third_party/chromiumos-ec/Makefile.app

```diff
@@ -0,0 +1,27 @@
+# Makefile for chromiumos-ec.a library. This includes code under third
+# party licenses.
+
+CHROMIUM_CFILES =  $(wildcard $(CHROMIUMOS_DIR)/*.c)
+CHROMIUM_OBJS   =  $(notdir $(patsubst %.c, %.o, $(CHROMIUM_CFILES)))
+
+define CHROMIUM_EC_RULES
+
+$$(BUILDDIR)/$(1)/chromiumos-ec:
+	$$(TRACEDIR)
+	$$(Q)mkdir -p $$@
+
+$(BUILDDIR)/$(1)/chromiumos-ec/sha256.o: $(CHROMIUMOS_DIR)/sha256.c | $$(BUILDDIR)/$(1)/chromiumos-ec
+#	echo $(BUILDDIR)
```

Comment:
This file looks like it written for this PR (i.e. it's not pulled from chromium-ec). In that case, please remove the commented-out debug lines (14, 20, and 27)

Reply:
Done.